src/openstudio_lib/OSWebEnginePage.cpp

```diff
@@ -42,8 +44,15 @@ OSUrlRequestInterceptor::OSUrlRequestInterceptor(QObject* parent) : QWebEngineUr
 OSUrlRequestInterceptor::~OSUrlRequestInterceptor() = default;
 
 void OSUrlRequestInterceptor::interceptRequest(QWebEngineUrlRequestInfo& info) {
+  static QByteArray userAgent;
+  if (userAgent.isEmpty()) {
+    userAgent.append("OpenStudioApp/");
+    userAgent.append(OpenStudioApplicationVersionMajor());
+    userAgent.append(".");
+    userAgent.append(OpenStudioApplicationVersionMinor());
+  }
   info.setHttpHeader("Accept-Language", "en-US,en;q=0.9,es;q=0.8,de;q=0.7");
-  info.setHttpHeader("Access-Control-Allow-Origin", "*");
+  info.setHttpHeader("User-Agent", userAgent);
```

Comment:
@jmarrec the openstreetmap website was mad at us for not setting a proper user agent per their usage guidelines: https://operations.osmfoundation.org/policies/nominatim/

Also, Access-Control-Allow-Origin is a response header and we were triggering CORS checking by including it, removing it means we don't care about CORS

Reply:
Hpp:

```
private:
  QByteArray userAgent;
```

ctor:
```
OSUrlRequestInterceptor::OSUrlRequestInterceptor(QObject* parent)
  : QWebEngineUrlRequestInterceptor(parent),
    userAgent(
      QByteArray::fromStdString(fmt::format("OpenStudioApp/{}.{}", OpenStudioApplicationVersionMajor(), OpenStudioApplicationVersionMinor()))) {}
```